docker/Dockerfile.i586-unknown-linux-gnu

```diff
@@ -11,12 +11,24 @@ COPY xargo.sh /
 RUN /xargo.sh
 
 RUN apt-get update && apt-get install --assume-yes --no-install-recommends \
-    g++-multilib
+    g++-i686-linux-gnu \
```

Comment:
i586 should actually continue to use multilib, since there is no native Linux image for it, nor support for this architecture anymore in Debian stable (it isn't even supported in oldoldstable, since it was no longer supported in stretch).